index.html

```diff
@@ -32,6 +32,7 @@
         <input type="button" id="userpresetnew" value="New" />
         <input type="button" id="userpresetdelete" value="Delete" disabled />
         <input type="button" id="userpresetexport" value="Save and Export" disabled />
+        <label> Volume </label><input type="checkbox" id="volume" checked="true" />
```

Comment:
"Volume" doesn't sound very accurate. Sound volume is usually changed on a scale, not toggled on or off.

How about "Mute", and it's unchecked by default?